### PR TITLE
add healthcheck to integration test tomcat startup

### PR DESCRIPTION
**Pull request by adamkorynta**
sometimes integration tests are executing faster than the war is deployed, resulting in a 404 exception

### CODE REVIEW

**Comment by adamkorynta** [comment]
@devinrei

File: cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java [hunk 1]
```diff
@@ -87,6 +89,30 @@ public void beforeAll(ExtensionContext context) throws Exception {
...
+    }
+
+    private static void healthCheck() throws InterruptedException {
+        int attempts = 0;
+        int maxAttempts = 15;
+        for (; attempts < maxAttempts; attempts++) {
+            try {
+                given()
+                .when()
+                    .get("/offices/")
```
Comment (MikeNeilson) [inline cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java]: if you put a specific office at the end of this, like `/offices/SPK` you can check for a 200 OK and the response should also be slightly faster.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Looks good. Just a minor suggest. (what I'm suggesting is actually my general goto to see if a given instance is behaving correctly as it requires a database connection.)

**Review by rma-rripken — DISMISSED** [review]

File: cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java (hunk 1 above)
Comment (adamkorynta) [reply cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java]: great point, updated

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**